src/content/roadmap/account-abstraction/index.md

```diff
@@ -0,0 +1,107 @@
+
+The way wallets work would also change under EIP-4337. Instead of each wallet re-implementing common but complex safety logic, those functions would be outsourced to a global wallet contract known as the "entry point". This would handle operations such as paying fees and executing EVM code so that wallet developers can focus on providing excellent user experiences.
+
+**Note** the EIP 4337 entry point contract was deployed to Ethereum Mainnet on 1st March 2023. You can see the contract on [Etherscan](https://etherscan.io/address/0x0576a174D229E3cFA37253523E645A78A0C91B57).
+
+</ExpandableCard>
+
+<ExpandableCard title="EIP-2938: changing the Ethereum protocol to support account abstraction" eventCategory="/roadmap/account-abstract" eventName="clicked EIP-2938: changing the Ethereum protocol to support account abstraction">
+
+EIP-2938 aims to update the Ethereum protocol by introducing a new transaction type, `AA_TX_TYPE` that includes three fields: `nonce`, `target` and `data`, where `nonce` is a transaction counter, `target` is the entry point contract address and `data` is EVM bytecode. To execute these transactions, two new instructions (known as opcodes) have to be added to the EVM: `NONCE` and `PAYGAS`. The `NONCE` opcode tracks the transaction sequence and `PAYGAS` calculates and withdraws the gas required to execute the transaction from the contract's balance. These new features allow Ethereum to support smart contract wallets natively as the necessary infrastructure is built in to Ethereum's protocol.
+
+Note that EIP-2938 is currently not active. The community is currently favoring EIP-4337 because it does not require changes to the protocol.
+
+</ExpandableCard>
+
+## Current progress {#current-progress}
+
+Smart contract wallets are already available, but more upgrades are required to make them as decentralized and permissionless as possible. EIP-4337 is a mature proposal that does not require any changes to Ethereum's protocol, so it is possible that this could be implemented quickly. However, upgrades that alter Ethereum's protocol are currently not in active development, so those changes may take much longer to ship. It is also possible that account abstraction is achieved well enough by EIP-4337 that no protocol changes are ever required.
```

Comment:
@jmcook1186 "current progress" information is critical information and should be also presented on the main subpages. I would suggest moving this out of the dev docs there on all pages or including in on both.

Reply:
ok i added current progress summaries to the main subpages - lmk what you think